libs/vmst-client/src/lib/vmst-client.module.ts

```diff
@@ -0,0 +1,42 @@
+import { Module, DynamicModule } from '@nestjs/common'
+import fetch from 'isomorphic-fetch'
+import {
+  Configuration,
+  ParentalLeaveApi,
+  PensionApi,
+  PregnancyApi,
+  UnionApi,
+} from '../../gen/fetch'
+
+export interface VMSTClientModuleConfig {
+  apiKey: string
+  xRoadPath: string
```

Comment:
I definitely go in circles if this should be just `basePath`, or if this module should be even more intelligent. I.e. only receive `xRoadBasePath` from env somehow and add the rest (member stuff, apiPath) automatically.

It could also make it easier to configure (fewer env variables).

Fine to keep it like this for now.